### PR TITLE
[Feature] append blocking conditions for `/txs` query with `--public` flag on LCD

### PR DESCRIPTION
**Pull request by YunSuk-Yeo**
## Summary of changes

* Remove custom `tx.height` interpretation
* Update swagger to apply new flags `tx.minheight` & `tx.maxheight`
* Enforce to use` tx.height` or (`tx.maxheight` & `tx.minheight`) events when rest-server is running with `--public` flag

## Report of required housekeeping

- [ ] Github issue OR spec proposal link
- [ ] Wrote tests
- [ ] Updated API documentation (client/lcd/swagger-ui/swagger.yaml)
- [ ] Added a relevant changelog entry: clog add [section] [stanza] [message]

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

File: x/auth/client/rest/query.go [hunk 1]
```diff
@@ -92,11 +86,6 @@ func QueryTxsRequestHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
-		//append tx height events
-		if txHeightEvents != nil && len(txHeightEvents) > 0 {
-			events = append(events, txHeightEvents...)
-		}
-
 		searchResult, err := utils.QueryTxsByEvents(cliCtx, events, page, limit)
```
Comment (hanjukim) [inline x/auth/client/rest/query.go:89]: where is code for appending events? It loooks to me that there is only validation.

Comment (YunSuk-Yeo) [reply x/auth/client/rest/query.go:89]: https://github.com/terra-project/core/blob/8a3751eb7fd4fde724556d77b7280d4f7b0e974d/x/auth/client/rest/query.go#L73

Now we don’t need to support our custom height options. Cosmos will do that with minheight and maxheight flag

Comment (YunSuk-Yeo) [reply x/auth/client/rest/query.go:89]: https://github.com/terra-project/core/blob/8a3751eb7fd4fde724556d77b7280d4f7b0e974d/x/auth/client/rest/query.go#L73

Within this function, minheight and maxheight and height will be interpreted to tendemint events.

Comment (YunSuk-Yeo) [reply x/auth/client/rest/query.go:89]: https://github.com/cosmos/cosmos-sdk/blob/2727c14183d05f5c31ca11f2dcd52b0aa683fadf/types/rest/rest.go#L220

**Review by hanjukim — APPROVED** [review]

**Pull request merged**